README.md

```diff
@@ -16,3 +28,7 @@ See [CONTRIBUTING](CONTRIBUTING.md) for details.
 Copyright © 2021, Office for National Statistics (https://www.ons.gov.uk)
 
 Released under MIT license, see [LICENSE](LICENSE.md) for details.
+
+[node]: <https://nodejs.org/en/>
+[npm]: <https://www.npmjs.com/>
+[sass-linter]: <https://onsdigital.github.io/ons-pattern-library-starter/>
```

Comment:
I think this should either be a different link or text that's something other than  'sass-linter'?

Reply:
Removed